exotic/api/filters.py

```diff
@@ -72,7 +75,5 @@
         "LCO Pan-STARRS Y": "PanSTARRS Y",
         "LCO Pan-STARRS w": "PanSTARRS w",
 
-        "Clear (unfiltered) reduced to V sequence": "MObs CV",
```

Comment:
I support this if we are allowing a more liberal custom filter arrangement. 

There's no cost to matching the name, though. If we want to maintain a different 'name' (e.g. abbreviation) I think we should approach that with a slightly different algorithm. ... We can talk about it tonight if you're free.